Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -3745,7 +3747,7 @@ def rtr_polling_retrieve_file_command(args: dict):
         if args.get('SHA256'):
             # the status is ready, we can get the extracted files
             args.pop('SHA256')
-            return rtr_get_extracted_file(get_status_response, args.get('fileName'))  # type:ignore
```

Comment:
Just to understand your changes better - was it a bug? or is there any special reason why we changed the key from fileName to filename?

Reply:
We changed the key name from fileName to filename because the argument name in the command arguments is defined as filename. Here we are trying to pass the same filename argument. The command wasn't saving the file to the custom filename given in the command as the argument was not being passed in the integration properly